.github/workflows/run-e2e-tests-and-push-node-to-ecr.yml

```diff
@@ -728,110 +553,71 @@ jobs:
         timeout-minutes: 10
         env:
           # Relative to local-tests/ directory
-          ALEPH_NODE_BINARY: aleph-release-node/aleph-node
+          ALEPH_NODE_BINARY: aleph-test-node/aleph-node
         run: ./.github/scripts/test_multiple_restarts.sh ${{ matrix.pruning }}
 
-  check-runtime-change:
-    name: Inspect whether runtime version has been changed (compared with main)
+  push-node-image-to-ecr:
+    name: Push production node docker image to the ECR
```

Comment:
Let's change it to `Push node docker image to ECR`.  That "production" word might suggest that we actually push something to production not that we push production-version of the node image.